README.md

```diff
@@ -8,378 +8,4 @@ The package strives to deliver on the following design goals:
-    val (i1, i2) = t
-    r <- applyMatrices(i1, i2, v)
-  } yield r
-```
-
-The `par` combinator has resource-safe semantics. If one computation fails, the other computation will be interrupted, to prevent wasting resources.
-
-### Racing
-
-Two `IO` actions can be *raced*, which means they will be executed in parallel, and the value of the first action that completes successfully will be returned.
-
-```scala
-action1.race(action2)
-```
-
-The `race` combinator is resource-safe, which means that if one of the two actions returns a value, the other one will be interrupted, to prevent wasting resources.
-
-The `race` and even `par` combinators are a specialization of a much-more powerful combinator called `raceWith`, which allows executing user-defined logic when the first of two actions succeeds.
-
-# Performance
-
-`scalaz.zio` has excellent performance, featuring a hand-optimized, low-level interpreter that achieves zero allocations for right-associated binds, and minimal allocations for left-associated binds.
-
-The `benchmarks` project may be used to compare `IO` with other effect monads, including `Future` (which is not an effect monad but is included for reference), Monix `Task`, and Cats `IO`.
-
-As of the time of this writing, `IO` is significantly faster than or at least comparable to all other purely functional solutions.
-
-# Thread Shifting - JVM
-
-By default, fibers make no guarantees as to which thread they execute on. They may shift between threads, especially as they execute for long periods of time.
-
-Fibers only ever shift onto the thread pool of the runtime system, which means that by default, fibers running for a sufficiently long time will always return to the runtime system's thread pool, even when their (asynchronous) resumptions were initiated from other threads.
-
-For performance reasons, fibers will attempt to execute on the same thread for a (configurable) minimum period, before yielding to other fibers. Fibers that resume from asynchronous callbacks will resume on the initiating thread, and continue for some time before yielding and resuming on the runtime thread pool.
-
-These defaults help guarantee stack safety and cooperative multitasking. They can be changed in `RTS` if automatic thread shifting is not desired.
-
-# ZIO in the Wild
-
-In an ideal world ZIO would be the one and only effect system, but we live in this world and more often than not we have to interact with code using other libraries. `zio-interop` is the module enabling the necessary integrations to achieve that in a seamless manner. Read the docs for more details: [Interop Docs](interop/README.md)
-
-If you're using SBT, add the following line to your build file:
-
-```scala
-libraryDependencies += "org.scalaz" %% "scalaz-zio-interop" % "0.1-SNAPSHOT"
-```
-
-# Legal
-
-Copyright (C) 2017-2018 John A. De Goes. All rights reserved.
+**Documentation** and **Examples** can be found on the [microsite](https://scalaz.github.io/scalaz-zio/).
```

Comment:
This is good. We should keep a copy of `Legal` here on the README, though.

Also I feel like the link could be better highlighted, something like:

See the ZIO microsite for documentation and examples:

## <a href="">ZIO Website</a>

This way, we'll draw user's attention to it.